src/js/pagination/paging-collection.js

```diff
@@ -46,8 +46,10 @@
             queryParams: {
                 currentPage: 'page',
                 pageSize: 'page_size',
+                totalRecords: 'count',
+                totalPages: 'num_pages',
```

Comment:
@dan-f yeah, they would be incompatible. But I know how to fix it, I will do my changes in that and tag you for review.